data/lic-mapping.json

```diff
@@ -20,6 +22,9 @@
       "Apache-2.0 OR MIT",
       "Apache2.0",
       "apache-2-0",
+      "APL2",
+      "the Apache License, ASL Version 2.0",
+      "Apache Publich License 2.0",
```

Comment:
Minor typo: Public**h**

Reply:
Could you send a new PR? It is possible that the typo is deliberate so that it matches with the correct spdx id but let's not implement stemming algorithms in cdxgen (or not yet ;))

Reply:
The typeo is deliberate, see https://central.sonatype.com/artifact/org.scalanlp/breeze_2.12/1.3?smo=true